student-store-ui/src/components/CheckoutForm/CheckoutForm.jsx

```diff
@@ -1,9 +1,55 @@
 import * as React from "react"
-import "./ShoppingCart.css"; 
+import "./CheckoutForm.css"; 
+
+export default function checkoutForm({isOpen, shoppingCart, checkoutForm, handleOnCheckoutFormChange, handleOnSubmitCheckoutForm, newOrder}){
+    let subtotal = 0.00;
 
-export default function checkoutForm(){
     return(
-        <div className="checkout-form">
+        <div className={isOpen ? "checkout-form": "hidden"}>
+            <div className="checkout-form-input">
+                <input className="input-field"
+                   type="text"
+                   name="name"
+                   placeholder="Student Name"
+                   value={checkoutForm.name}
+                   onChange={(e) => handleOnCheckoutFormChange(e.target.name, e.target.value)}
+                />
+                <input className="input-field"
+                   type="email"
+                   name="email"
+                   placeholder="student@codepath.org"
+                   value={checkoutForm.email}
+                   onChange={(e) => handleOnCheckoutFormChange(e.target.name, e.target.value)}
+                />
+            </div>
+
+            <div className="checkout">
+                <button className="checkout-button" onClick={(e) => handleOnSubmitCheckoutForm()}>
+                    Checkout
+                </button>
+            </div>
+            {
+                Object.keys(newOrder).length > 0 &&
+                <div className="recipt"> 
+                    <p className="recipt-name">Receipt</p>
+                    <p>Showing recipt for {newOrder.name} avilable at {newOrder.email}</p>
+                    {Object.entries(newOrder.shoppingCart).map(([productName, productInfo]) => {
```

Comment:
nit: this part looks messy. This can be a array that is setup before JSX starts, its own component or a function that returns a array.